replication/event.go

```diff
@@ -229,15 +229,59 @@ type PreviousGTIDsEvent struct {
 	GTIDSets string
 }
 
+type GtidFormat int
+
+const (
+	GtidFormatClassic = iota
+	GtidFormatTagged
+)
+
+func decodeSid(data []byte) (format GtidFormat, sidnr uint64) {
```

Comment:
can you put the referenced MySQL source code or document in comment, so other reviewers can double check the implementation?

Reply:
done